include/zephyr/mgmt/mcumgr/smp.h

```diff
@@ -13,21 +13,46 @@
 extern "C" {
 #endif
 
+/**
+ * @brief SMP transport out function flags
+ *
+ * @details Flags passed to the transport out function
+ */
+enum smp_transport_out_flags {
+	SMP_TRANSPORT_OUT_NONE = 0,
+	SMP_TRANSPORT_OUT_FIRST_PACKET = BIT(0),
+	SMP_TRANSPORT_OUT_LAST_PACKET = BIT(1),
+};
+
```

Comment:
I do not think this is needed: when you store `total_len` and `sent_bytes` in your "fragmentation state/context" `zephyr_tx_information`, you have all the info needed to figure where you are at a transfer.

Reply:
Removed